docs/dev/zksync-v2/blocks-and-time.md

```diff
@@ -1,3 +1,24 @@
 # Block numbers and time
 
-TODO.
+On ZkSync there are two types of blocks: L2 blocks and L1 batches.
+
+## L2 blocks
+
+L2 block is similar to a common block on L1. L2 block time is almost constant, like a few seconds.
```

Comment:
What is a "common block"? Does "uncommon block" exist on L1?
"Almost constant, like a few seconds" sounds very uncertain -- is it a formal definition? Why can't it be "fully constant"?